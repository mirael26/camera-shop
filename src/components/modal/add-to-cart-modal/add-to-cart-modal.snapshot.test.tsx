import { renderWithReduxAndRouter } from '../../../test/helpers/render-with-redux-and-router';
import { productMock } from '../../../test/mocks';
import AddToCartModal from './add-to-cart-modal';

jest.mock('./add-to-cart-modal-content/add-to-cart-modal-content', () => 'AddToCartModalContent');

test('AddToCartModal renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<AddToCartModal/>, { initialState: {
    state: {
      addToCartModalOpen: true,
      addingToCartItem: productMock,
    }
  }});
  expect(asFragment()).toMatchSnapshot();
});
