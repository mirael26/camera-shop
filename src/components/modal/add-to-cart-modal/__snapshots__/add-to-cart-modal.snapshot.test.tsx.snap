// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddToCartModal renders correctly 1`] = `
<DocumentFragment>
  <div
    class="modal is-active"
    data-testid="add-to-cart-modal"
  >
    <addtocartmodalcontent
      product="[object Object]"
    />
  </div>
</DocumentFragment>
`;
